test/integration/oauth_request_header_test.go

```diff
@@ -89,6 +95,11 @@ qLwYJxjzwYTLvLYPU5vHmdg8v5wIXh0TaRTDTdKViISGD09aiXSYzw==
 )
 
 func TestOAuthRequestHeader(t *testing.T) {
+	// Test data used by auth proxy
```

Comment:
I'm trying to batch integration tests where it makes sense. The check for the redirect and testing the login command against a server configured this way didn't seem like too much to add